### ISSUE TITLE
Update Japanese translation

### ISSUE THREAD

**Issue by Yz2house** [Beep6581/RawTherapee#5074]
translated a new string and some modifications
rgds
[Japanese.txt](https://github.com/Beep6581/RawTherapee/files/2659407/Japanese.txt)

**Comment by Beep6581** [comment]
Hey @Yz2house 

Sorry, it seems I overlooked this issue when tagging rc2 just now.
Here is a diff of your changes from the file attached above against `5.5-rc2`:
```diff
diff --git a/rtdata/languages/Japanese b/rtdata/languages/Japanese
index 683f9f24b..66d964bec 100644
--- a/rtdata/languages/Japanese
+++ b/rtdata/languages/Japanese
@@ -254,6 +254,7 @@ GENERAL_AUTO;自動
 GENERAL_BEFORE;補正前
 GENERAL_CANCEL;キャンセル
 GENERAL_CLOSE;閉じる
+GENERAL_CURRENT;現在
 GENERAL_DISABLE;無効
 GENERAL_DISABLED;無効
 GENERAL_ENABLE;有効
@@ -757,7 +758,7 @@ HISTORY_MSG_485;レンズ補正
 HISTORY_MSG_486;レンズ補正 - カメラ
 HISTORY_MSG_487;レンズ補正 - レンズ
 HISTORY_MSG_488;ダイナミックレンジ圧縮
-HISTORY_MSG_489;DRC - 細部
+HISTORY_MSG_489;DRC - ディテール
 HISTORY_MSG_490;DRC - 量
 HISTORY_MSG_491;ホワイトバランス
 HISTORY_MSG_492;RGBカーブ
@@ -2090,7 +2091,7 @@ TP_SOFTLIGHT_STRENGTH;強さ
 TP_TM_FATTAL_AMOUNT;量
 TP_TM_FATTAL_ANCHOR;アンカー
 TP_TM_FATTAL_LABEL;ダイナミックレンジ圧縮
-TP_TM_FATTAL_THRESHOLD;細部
+TP_TM_FATTAL_THRESHOLD;ディテール
 TP_VIBRANCE_AVOIDCOLORSHIFT;色ずれを回避
 TP_VIBRANCE_CURVEEDITOR_SKINTONES;HH
 TP_VIBRANCE_CURVEEDITOR_SKINTONES_LABEL;肌色トーン
@@ -2340,4 +2341,3 @@ ZOOMPANEL_ZOOMOUT;ズームアウト\nショートカット: <b>-</b>
 !!!!!!!!!!!!!!!!!!!!!!!!!

 !CURVEEDITOR_CATMULLROM;Flexible
-!GENERAL_CURRENT;Current
```

There are two keys left to be translated. If you translate them and incorporate these three overlooked changes, we can include that for the final 5.5 release this weekend.
As always, the latest file is here:
https://github.com/Beep6581/RawTherapee/blob/dev/rtdata/languages/Japanese

**Comment by Beep6581** [comment]
Closed by 836c44bf93aa0f7c8e7123a8d599b7cc334e4eb4
